### ISSUE TITLE
Mod causes multiplayer clients to think it's missing certain mods

### ISSUE THREAD

**Issue by LordVGames** [Goorakh/AudioOverlapFix#1]
I added the mod to a pack of mine that I play with a friend and noticed that, when attempting to join my friend, it would say 1 mod was incompatible, and another was required, even though both were loaded in our games, and with the correct versions as well. Didn't matter if I joined him or if he joined me, the one joining would get the same exact message about mods being missing/incompatible. This stopped after disabling the AudioOverlapFix mod.

Not sure yet if a specific mod is causing this to happen in conjunction with AudioOverlapFix, or if it's just the fault of AudioOverlapFix.

**Comment by Goorakh** [comment]
Not much I can tell from just a description, please attach screenshot(s) of the connection error message, and log files for both you and your friend after failing to connect.

**Comment by LordVGames** [comment]
So after I had uninstalled the mod, I reinstalled it recently (before 1.2.0) and was able to play just fine, no error message. Makes me think the mod itself wasn't causing it. Likely something went wrong somewhere with r2modman and messed things up, with this mod being related to it. I hadn't seen r2modman mess up like that before, so that's why I made an issue.

Nonetheless, I'll still leave the error message we got here. If it somehow happens again, and this mod is the culprit, I'll comment on this issue.
![image](https://github.com/Goorakh/AudioOverlapFix/assets/51967559/3fdf7a0e-c743-4539-aeed-b01089d94f1f)
